metrics/influxdb.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""InfluxDB client that pushes time series points from JSON."""
+
+import httplib
+import json
+import time
+import urllib
+
+class Point(object):
+    """Represents an InfluxDB time series point and handles serialization."""
```

Comment:
Will you link to the documentation you used to figure out an InfluxDB time series point format?